indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -239,6 +249,7 @@
             :short-name.lowercase (util/safe-lowercase short-name)
             :version-id version-id
             :version-id.lowercase (util/safe-lowercase version-id)
+            :parsed-version-id.lowercase (util/safe-lowercase parsed-version-id)
```

Comment:
Inserting dots in these keys seems an odd convention to me ... like JavaScript-isms making it into Clojure code. Why not `:doi-lowercase`, `:version-id-lowercase` and `:parsed-version-id-lowercase` instead?

Is there something else consuming these that we don't control, that expects this?